drivers/issi/is31fl3741.h

```diff
@@ -53,3 +50,373 @@ void IS31FL3741_update_led_control_registers(uint8_t addr1, uint8_t addr2);
+#define CS17_SW8 0xE2
+#define CS18_SW8 0xE3
+#define CS19_SW8 0xE4
+#define CS20_SW8 0xE5
+#define CS21_SW8 0xE6
+#define CS22_SW8 0xE7
+#define CS23_SW8 0xE8
+#define CS24_SW8 0xE9
+#define CS25_SW8 0xEA
+#define CS26_SW8 0xEB
+#define CS27_SW8 0xEC
+#define CS28_SW8 0xED
+#define CS29_SW8 0xEE
+#define CS30_SW8 0xEF
+
+#define CS1_SW9  0xF0
+#define CS2_SW9  0xF1
+#define CS3_SW9  0xF2
+#define CS4_SW9  0xF3
+#define CS5_SW9  0xF4
+#define CS6_SW9  0xF5
+#define CS7_SW9  0xF6
+#define CS8_SW9  0xF7
+#define CS9_SW9  0xF8
+#define CS10_SW9 0xF9
+#define CS11_SW9 0xFA
+#define CS12_SW9 0xFB
+#define CS13_SW9 0xFC
+#define CS14_SW9 0xFD
+#define CS15_SW9 0xFE
+#define CS16_SW9 0xFF
+#define CS17_SW9 0x100
+#define CS18_SW9 0x101
+#define CS19_SW9 0x102
+#define CS20_SW9 0x103
+#define CS21_SW9 0x104
+#define CS22_SW9 0x105
+#define CS23_SW9 0x106
+#define CS24_SW9 0x107
+#define CS25_SW9 0x108
+#define CS26_SW9 0x109
+#define CS27_SW9 0x10A
+#define CS28_SW9 0x10B
+#define CS29_SW9 0x10C
+#define CS30_SW9 0x10D
+
+#define CS31_SW1 0x10E
+#define CS32_SW1 0x10F
+#define CS33_SW1 0x110
+#define CS34_SW1 0x111
+#define CS35_SW1 0x112
+#define CS36_SW1 0x113
+#define CS37_SW1 0x114
+#define CS38_SW1 0x115
+#define CS39_SW1 0x116
+
+#define CS31_SW2 0x117
+#define CS32_SW2 0x118
+#define CS33_SW2 0x119
+#define CS34_SW2 0x11A
+#define CS35_SW2 0x11B
+#define CS36_SW2 0x11C
+#define CS37_SW2 0x11D
+#define CS38_SW2 0x11E
+#define CS39_SW2 0x11F
+
+#define CS31_SW3 0x120
+#define CS32_SW3 0x121
+#define CS33_SW3 0x122
+#define CS34_SW3 0x123
+#define CS35_SW3 0x124
+#define CS36_SW3 0x125
+#define CS37_SW3 0x126
+#define CS38_SW3 0x127
+#define CS39_SW3 0x128
+
+#define CS31_SW4 0x129
+#define CS32_SW4 0x12A
+#define CS33_SW4 0x12B
+#define CS34_SW4 0x12C
+#define CS35_SW4 0x12D
+#define CS36_SW4 0x12E
+#define CS37_SW4 0x12F
+#define CS38_SW4 0x130
+#define CS39_SW4 0x131
+
+#define CS31_SW5 0x132
+#define CS32_SW5 0x133
+#define CS33_SW5 0x134
+#define CS34_SW5 0x135
+#define CS35_SW5 0x136
+#define CS36_SW5 0x137
+#define CS37_SW5 0x138
+#define CS38_SW5 0x139
+#define CS39_SW5 0x13A
+
+#define CS31_SW6 0x13B
+#define CS32_SW6 0x13C
+#define CS33_SW6 0x13D
+#define CS34_SW6 0x13E
+#define CS35_SW6 0x13F
+#define CS36_SW6 0x140
+#define CS37_SW6 0x141
+#define CS38_SW6 0x142
+#define CS39_SW6 0x143
+
+#define CS31_SW7 0x144
+#define CS32_SW7 0x145
+#define CS33_SW7 0x146
+#define CS34_SW7 0x147
+#define CS35_SW7 0x148
+#define CS36_SW7 0x149
+#define CS37_SW7 0x14A
+#define CS38_SW7 0x14B
+#define CS39_SW7 0x14C
+
+#define CS31_SW8 0x14D
+#define CS32_SW8 0x14E
+#define CS33_SW8 0x14F
+#define CS34_SW8 0x150
+#define CS35_SW8 0x151
+#define CS36_SW8 0x152
+#define CS37_SW8 0x153
+#define CS38_SW8 0x154
+#define CS39_SW8 0x155
+
+#define CS31_SW9 0x156
+#define CS32_SW9 0x157
+#define CS33_SW9 0x158
+#define CS34_SW9 0x159
+#define CS35_SW9 0x15A
+#define CS36_SW9 0x15B
+#define CS37_SW9 0x15C
+#define CS38_SW9 0x15D
+#define CS39_SW9 0x15E
+
```

Comment:
```suggestion
#define A_1 0x00
#define A_2 0x01
#define A_3 0x02
#define A_4 0x03
#define A_5 0x04
#define A_6 0x05
#define A_7 0x06
#define A_8 0x07
#define A_9 0x08
#define A_10 0x09
#define A_11 0x0A
#define A_12 0x0B
#define A_13 0x0C
#define A_14 0x0D
#define A_15 0x0E
#define A_16 0x0F
#define A_17 0x10
#define A_18 0x11
#define A_19 0x12
#define A_20 0x13
#define A_21 0x14
#define A_22 0x15
#define A_23 0x16
#define A_24 0x17
#define A_25 0x18
#define A_26 0x19
#define A_27 0x1A
#define A_28 0x1B
#define A_29 0x1C
#define A_30 0x1D
#define A_31 0x15A
#define A_32 0x15B
#define A_33 0x15C
#define A_34 0x15D
#define A_35 0x15E
#define A_36 0x15F
#define A_37 0x160
#define A_38 0x161
#define A_39 0x162

#define B_1 0x1E
#define B_2 0x1F
#define B_3 0x20
#define B_4 0x21
#define B_5 0x22
#define B_6 0x23
#define B_7 0x24
#define B_8 0x25
#define B_9 0x26
#define B_10 0x27
#define B_11 0x28
#define B_12 0x29
#define B_13 0x2A
#define B_14 0x2B
#define B_15 0x2C
#define B_16 0x2D
#define B_17 0x2E
#define B_18 0x2F
#define B_19 0x30
#define B_20 0x31
#define B_21 0x32
#define B_22 0x33
#define B_23 0x34
#define B_24 0x35
#define B_25 0x36
#define B_26 0x37
#define B_27 0x38
#define B_28 0x39
#define B_29 0x3A
#define B_30 0x3B
#define B_31 0x163
#define B_32 0x164
#define B_33 0x165
#define B_34 0x166
#define B_35 0x167
#define B_36 0x168
#define B_37 0x169
#define B_38 0x16A
#define B_39 0x16B

#define C_1 0x3C
#define C_2 0x3D
#define C_3 0x3E
#define C_4 0x3F
#define C_5 0x40
#define C_6 0x41
#define C_7 0x42
#define C_8 0x43
#define C_9 0x44
#define C_10 0x45
#define C_11 0x46
#define C_12 0x47
#define C_13 0x48
#define C_14 0x49
#define C_15 0x4A
#define C_16 0x4B
#define C_17 0x4C
#define C_18 0x4D
#define C_19 0x4E
#define C_20 0x4F
#define C_21 0x50
#define C_22 0x51
#define C_23 0x52
#define C_24 0x53
#define C_25 0x54
#define C_26 0x55
#define C_27 0x56
#define C_28 0x57
#define C_29 0x58
#define C_30 0x59
#define C_31 0x16C
#define C_32 0x16D
#define C_33 0x16E
#define C_34 0x16F
#define C_35 0x170
#define C_36 0x171
#define C_37 0x172
#define C_38 0x173
#define C_39 0x174

#define D_1 0x5A
#define D_2 0x5B
#define D_3 0x5C
#define D_4 0x5D
#define D_5 0x5E
#define D_6 0x5F
#define D_7 0x60
#define D_8 0x61
#define D_9 0x62
#define D_10 0x63
#define D_11 0x64
#define D_12 0x65
#define D_13 0x66
#define D_14 0x67
#define D_15 0x68
#define D_16 0x69
#define D_17 0x6A
#define D_18 0x6B
#define D_19 0x6C
#define D_20 0x6D
#define D_21 0x6E
#define D_22 0x6F
#define D_23 0x70
#define D_24 0x71
#define D_25 0x72
#define D_26 0x73
#define D_27 0x74
#define D_28 0x75
#define D_29 0x76
#define D_30 0x77
#define D_31 0x175
#define D_32 0x176
#define D_33 0x177
#define D_34 0x178
#define D_35 0x179
#define D_36 0x17A
#define D_37 0x17B
#define D_38 0x17C
#define D_39 0x17D

#define E_1 0x78
#define E_2 0x79
#define E_3 0x7A
#define E_4 0x7B
#define E_5 0x7C
#define E_6 0x7D
#define E_7 0x7E
#define E_8 0x7F
#define E_9 0x80
#define E_10 0x81
#define E_11 0x82
#define E_12 0x83
#define E_13 0x84
#define E_14 0x85
#define E_15 0x86
#define E_16 0x87
#define E_17 0x88
#define E_18 0x89
#define E_19 0x8A
#define E_20 0x8B
#define E_21 0x8C
#define E_22 0x8D
#define E_23 0x8E
#define E_24 0x8F
#define E_25 0x90
#define E_26 0x91
#define E_27 0x92
#define E_28 0x93
#define E_29 0x94
#define E_30 0x95
#define E_31 0x17E
#define E_32 0x17F
#define E_33 0x180
#define E_34 0x181
#define E_35 0x182
#define E_36 0x183
#define E_37 0x184
#define E_38 0x185
#define E_39 0x186

#define F_1 0x96
#define F_2 0x97
#define F_3 0x98
#define F_4 0x99
#define F_5 0x9A
#define F_6 0x9B
#define F_7 0x9C
#define F_8 0x9D
#define F_9 0x9E
#define F_10 0x9F
#define F_11 0xA0
#define F_12 0xA1
#define F_13 0xA2
#define F_14 0xA3
#define F_15 0xA4
#define F_16 0xA5
#define F_17 0xA6
#define F_18 0xA7
#define F_19 0xA8
#define F_20 0xA9
#define F_21 0xAA
#define F_22 0xAB
#define F_23 0xAC
#define F_24 0xAD
#define F_25 0xAE
#define F_26 0xAF
#define F_27 0xB0
#define F_28 0xB1
#define F_29 0xB2
#define F_30 0xB3
#define F_31 0x187
#define F_32 0x188
#define F_33 0x189
#define F_34 0x18A
#define F_35 0x18B
#define F_36 0x18C
#define F_37 0x18D
#define F_38 0x18E
#define F_39 0x18F

#define G_1 0x100
#define G_2 0x101
#define G_3 0x102
#define G_4 0x103
#define G_5 0x104
#define G_6 0x105
#define G_7 0x106
#define G_8 0x107
#define G_9 0x108
#define G_10 0x109
#define G_11 0x10A
#define G_12 0x10B
#define G_13 0x10C
#define G_14 0x10D
#define G_15 0x10E
#define G_16 0x10F
#define G_17 0x110
#define G_18 0x111
#define G_19 0x112
#define G_20 0x113
#define G_21 0x114
#define G_22 0x115
#define G_23 0x116
#define G_24 0x117
#define G_25 0x118
#define G_26 0x119
#define G_27 0x11A
#define G_28 0x11B
#define G_29 0x11C
#define G_30 0x11D
#define G_31 0x190
#define G_32 0x191
#define G_33 0x192
#define G_34 0x193
#define G_35 0x194
#define G_36 0x195
#define G_37 0x196
#define G_38 0x197
#define G_39 0x198

#define H_1 0x11E
#define H_2 0x11F
#define H_3 0x120
#define H_4 0x121
#define H_5 0x122
#define H_6 0x123
#define H_7 0x124
#define H_8 0x125
#define H_9 0x126
#define H_10 0x127
#define H_11 0x128
#define H_12 0x129
#define H_13 0x12A
#define H_14 0x12B
#define H_15 0x12C
#define H_16 0x12D
#define H_17 0x12E
#define H_18 0x12F
#define H_19 0x130
#define H_20 0x131
#define H_21 0x132
#define H_22 0x133
#define H_23 0x134
#define H_24 0x135
#define H_25 0x136
#define H_26 0x137
#define H_27 0x138
#define H_28 0x139
#define H_29 0x13A
#define H_30 0x13B
#define H_31 0x199
#define H_32 0x19A
#define H_33 0x19B
#define H_34 0x19C
#define H_35 0x19D
#define H_36 0x19E
#define H_37 0x19F
#define H_38 0x1A0
#define H_39 0x1A1

#define I_1 0x13C
#define I_2 0x13D
#define I_3 0x13E
#define I_4 0x13F
#define I_5 0x140
#define I_6 0x141
#define I_7 0x142
#define I_8 0x143
#define I_9 0x144
#define I_10 0x145
#define I_11 0x146
#define I_12 0x147
#define I_13 0x148
#define I_14 0x149
#define I_15 0x14A
#define I_16 0x14B
#define I_17 0x14C
#define I_18 0x14D
#define I_19 0x14E
#define I_20 0x14F
#define I_21 0x150
#define I_22 0x151
#define I_23 0x152
#define I_24 0x153
#define I_25 0x154
#define I_26 0x155
#define I_27 0x156
#define I_28 0x157
#define I_29 0x158
#define I_30 0x159
#define I_31 0x1A2
#define I_32 0x1A3
#define I_33 0x1A4
#define I_34 0x1A5
#define I_35 0x1A6
#define I_36 0x1A7
#define I_37 0x1A8
#define I_38 0x1A9
#define I_39 0x1AA
```

Reply:
Please check the DataSheet of ISSI3741. CS and SW are from the datasheet pins.

Reply:
Yes, but we should use the letter_number notation as with the other drivers.

Reply:
> Yes, but we should use the letter_number notation as with the other drivers.

Sorry about that. I don't think so. cause the other issi drivers follow their datasheet. so I think ISSI3741 should be also.
if we change it to letter_number will enhance the difficult for the developer who select ISSI3741.

Reply:
That's a fair point, and now that I think about it this notation might be better for the other drivers too.